### ISSUE TITLE
supply whatever e-pubmed requires to make link outs to PomBase

### ISSUE THREAD

**Issue by ValWood** [pombase/pombase-chado#560]
Jo McEntire:
It would be great to get some links from Europe PMC to PomBase as we have done for Worm and FlyBase. It is very straightforward:
http://europepmc.org/LabsLink
I have cc’ed our Author support staff who can give you some help with this should you need it.
helpdesk@EuropePMC.org

It’s a bit cryptic, but this search gets you Wormbase-linked records:
http://europepmc.org/search?query=%28LABS_PUBS:%221386%22%29
And this gets you FlyBase:
http://europepmc.org/search?query=%28LABS_PUBS:%221452%22%29
When you view the abstract, the “External Links” tab will contain the link. As I mentioned, we are reviewing our interface to surface these things more clearly.

**Comment by ValWood** [comment]
I'll plug an example in here shortly....

**Comment by ValWood** [comment]
Will depend on #558

**Comment by ValWood** [comment]
Jo:
yes, you can link from Europe PMC to any page you think is sensible/useful. Gene pages sound good to me ...

**Comment by ValWood** [comment]
not super -urgent but should increase PomBase accesses, so nice if quick and easy

**Comment by ValWood** [comment]
I'm in contact with science direct, call pending to find out more

**Comment by ValWood** [comment]
I'll call this low priority for now.
We can increase the web traffic on the new pages.

**Comment by ValWood** [comment]
replaced by 
https://github.com/pombase/website/issues/160
